### DISCUSSION TITLE
Sidenav change tooltip color on hover

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2555] [Q&A]
![Снимок экрана (13)](https://user-images.githubusercontent.com/78722918/174633321-250b6658-1df4-4c38-a6bf-1cd1941c0dfd.png)

I am using Sidenav and i want to change background color of tooltip which appears on hover

**Comment by SevenOutman** [comment]
I think some `renderTooltip` prop of `<Sidenav>` that allows fully customizing the tooltip element would be useful.

**Reply by abdullaev388** [reply]
There is no such prop

**Reply by SevenOutman** [reply]
No there isn't. I meant a new prop.

**Reply by abdullaev388** [reply]
so can you add it?
